src/braket/circuits/circuit.py

```diff
@@ -84,13 +84,15 @@ class Circuit:
     _ALL_QUBITS = "ALL"  # Flag to indicate all qubits in _qubit_observable_mapping
 
     @classmethod
-    def register_subroutine(cls, func: SubroutineCallable) -> None:
+    def register_subroutine(cls, func: SubroutineCallable, cls_docstring: str) -> None:
```

Comment:
Perhaps this should be `str | None` and default to `None`? For backward compatibility reasons it seems like it needs a default value.